### PR TITLE
Add simple figure style for content class

### PR DESCRIPTION
**Pull request by piojanu**
### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

It adds simple figure style so this gohugo shortcode:  
`{{< figure src="/images/N90.jpg" caption="N90 nebula, \"New stars shed light on the past\" by ESA/Hubble" >}}
`
produces this result:  
screenshot 2018-12-27 at 13 37 11

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
  Change content's css. 
- [x] Explain why and how the changes were necessary and implemented respectively
  Before it using gohugo shortcode for figures was giving bad formatted/bad looking figures with caption.
- [ ] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

**Comment by luizdepra** [comment]
Nice, thank you for the PR.

But, could you add the figure snippet into `exampleSite/content/posts/theme-demo.md`?

**Comment by piojanu** [comment]
Okay! Should I build the example website somehow?

**Comment by artspb** [comment]
I'm really sorry to pop-up in the middle of the conversation. But is it really necessary to use a 9 MB image to show the feature? I see the following downsides:

1. Once added to git it'll never be removed. Everyone who wants to use this wonderful theme will download it as a part of the example site.
2. Loading of the example site may take a while if the Internet connection is not stable. For mobile devises it additionally means draining of an accu.
3. I'm probably missing something but it looks like that the same demonstration effect can be achieve with say a 300 KB picture.

Once again, I'm sorry for bothering you with my comment. I understand that my concerns may be irrelevant for many if not all. I also like the feature and will definitely use it, thank you for implementing it.

**Comment by piojanu** [comment]
@artspb you're making really good point. I'll compress the image.

File: assets/scss/_content.scss [hunk 1]
```diff
@@ -39,6 +39,17 @@
       }
     }
   }
+  figure {
+    border: thin silver solid;
```
Comment (luizdepra) [inline assets/scss/_content.scss]: Just to keep styling consistent... It looks better without the border. So, please, remove this line.
And also change `margin` and `padding` to `0`.

File: assets/scss/_content.scss [hunk 2]
```diff
@@ -39,6 +39,17 @@
...
+    margin: 0.5em;
+    padding: 0.5em;
+  }
+  figcaption p {
+    text-align: center;
+    font-style: italic;
+    font-size: smaller;
```
Comment (luizdepra) [inline assets/scss/_content.scss]: Better use `1.6rem` as the `font-size` here.

**Comment by luizdepra** [comment]
I just request two minor changes. You will need to rebase yout fork and regenerate the resources too.

**Comment by piojanu** [comment]
@luizdepra Done :)

**Comment by cep21** [comment]
I noticed the example `{{< figure src="/images/` is no longer on the main branch.  Is figure still supported?  I'm looking for the best way to show images with captions.

**Comment by luizdepra** [comment]
@cep21, [`figure`](https://gohugo.io/content-management/shortcodes/#figure) comes from Hugo. So, it should work fine. Please tell me more about why it is not working, or provide an example. Or better, filling an issue.

**Comment by cep21** [comment]
Sorry: I did not discover a feature.  But there used to be a page that demonstrated how to use different things with the coder theme: `exampleSite/content/posts/theme-demo.md`.  That page, and the reference to `figure` above no longer exist.  Often, people remove demonstrations of things they don't intend others to use or support: so I asked here.

**Comment by luizdepra** [comment]
Hmm... I think it was removed accidentally. It should be [here](https://hugo-coder.netlify.app/posts/markdown-syntax/).

**Pull request merged**